packages/gasket-nextjs/package.json

```diff
@@ -13,7 +13,7 @@
     "test:watch": "npm run test:runner -- --watch",
     "test:coverage": "nyc --reporter=text --reporter=json-summary npm run test:runner",
     "posttest": "npm run lint",
-    "build": "babel src -d lib",
+    "build": "rimraf lib && babel src -d lib",
```

Comment:
i noticed the deployed package had some old artifacts from prior iterations of the plugin. Added this to remove the old ones before deployment.